src/loadConfig.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path')
+const fs = require('fs')
+const YAML = require('yaml')
+
+const CONFIG_FILE_OVERRIDE = 'custom/config.yaml'
+const CONFIG_FILE = 'config.yaml'
```

Comment:
We already have `.travis.yml`. Let's stick to one extension

Reply:
Mm, but I kind of like more yaml, it's more official. I don't see an issue
https://stackoverflow.com/questions/22268952/what-is-the-difference-between-yaml-and-yml-extension

If ir bothers u , I'll change it
Not sure if travis works with `yaml`

Reply:
Ok, stick to `yaml` if you wish

Reply:
`.ini` or `.ion` is out of the question, right? or using `.php` but writing a yaml